### PR TITLE
More future-proof comment kind

### PR DESCRIPTION
**Pull request by matklad**

### CODE REVIEW

**Comment by matklad** [comment]
bors r+
🤖

**Comment by DJMcNab** [comment]
/** is there twice - is that intended?

**Comment by matklad** [comment]
Nope, an excellent catch, thanks!

File: crates/ra_syntax/src/ast/tokens.rs [hunk 1]
```diff
@@ -21,52 +21,71 @@ impl<'a> AstToken<'a> for Comment<'a> {
...
+    Inner,
+    Outer,
+}
+
+const COMMENT_PREFIX_TO_KIND: &[(&str, CommentKind)] = {
+    use {CommentShape::*, CommentPlacement::*};
+    &[
+        ("///", CommentKind { shape: Line, doc: Some(Outer) }),
+        ("//!", CommentKind { shape: Line, doc: Some(Inner) }),
+        ("/**", CommentKind { shape: Block, doc: Some(Outer) }),
```
Comment (bjorn3) [inline crates/ra_syntax/src/ast/tokens.rs]: I think this should be `/*!`.

**Pull request merged**